spec/statesman/adapters/active_record_queries_spec.rb

```diff
@@ -68,7 +75,7 @@ def self.initial_state
 
         it { is_expected.to include model }
         it { is_expected.not_to include other_model }
-        its(:to_sql) { is_expected.to include('most_recent') }
+        its(:to_sql) { is_expected.to include('most_recent =') }
```

Comment:
I thought so too. but I realized that this spec returned true at all times.
```
$ git checkout 7fd6989 # my commit
$ bundle exec rake
/Users/otsuki.shinsuke/.rbenv/versions/2.2.1/bin/ruby -I/Users/otsuki.shinsuke/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rspec-core-3.3.1/lib:/Users/otsuki.shinsuke/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/rspec-support-3.3.0/lib /Users/otsuki.shinsuke/.rbenv/v
ersions/2.2.1/lib/ruby/gems/2.2.0/gems/rspec-core-3.3.1/exe/rspec --pattern spec/\*\*\{,/\*/\*\*\}/\*_spec.rb
Running with database adapter 'SQLite'

Randomized with seed 23601
................................................................................................................................................................................................................................................................................
.......F....F.............

Failures:

  1) Statesman::Adapters::ActiveRecordQueries with a most_recent column .in_state given a single state to_sql should include "most_recent ="
     Failure/Error: its(:to_sql) { is_expected.to include('most_recent =') }
       expected "SELECT \"my_active_record_models\".* FROM \"my_active_record_models\" LEFT OUTER JOIN my_active_record_model_transitions most_recent_transition\n             ON most_recent_transition.my_active_record_model_id =\n                  my_active_record_models.
id LEFT OUTER JOIN my_active_record_model_transitions other_transition\n             ON other_transition.my_active_record_model_id =\n                  my_active_record_models.id\n             AND other_transition.sort_key >\n                   most_recent_transition.sort
_key WHERE (most_recent_transition.to_state IN ('succeeded') AND most_recent_transition.to_state IS NOT NULL) AND \"other_transition\".\"id\" IS NULL" to include "most_recent ="
       Diff:
       @@ -1,2 +1,8 @@
       -most_recent =
       +SELECT "my_active_record_models".* FROM "my_active_record_models" LEFT OUTER JOIN my_active_record_model_transitions most_recent_transition
       +             ON most_recent_transition.my_active_record_model_id =
       +                  my_active_record_models.id LEFT OUTER JOIN my_active_record_model_transitions other_transition
       +             ON other_transition.my_active_record_model_id =
       +                  my_active_record_models.id
       +             AND other_transition.sort_key >
       +                   most_recent_transition.sort_key WHERE (most_recent_transition.to_state IN ('succeeded') AND most_recent_transition.to_state IS NOT NULL) AND "other_transition"."id" IS NULL
     # ./spec/statesman/adapters/active_record_queries_spec.rb:78:in `block (5 levels) in <top (required)>'
```
So `most_recent` is not only column name but also table alias name, `most_recent_transition` is included `most_recent`.
However this changing is strange. any idea?